packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,110 @@
+import { Notice } from "@cerner/terra-docs";
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import { Badge } from 'terra-icon/package.json?dev-site-package';
+
+<!-- Examples -->
+import DecorativeIconExample from './example/DecorativeIconExample?dev-site-example';
+import GroupOfIconsExample from './example/GroupOfIconsExample?dev-site-example';
+import InformativeIconExamples from './example/InformativeIconExamples?dev-site-example';
+import FunctionalIconExample from './example/FunctionalIconExample?dev-site-example';
+
+<!-- Icons used in the examples, but also I want to use them in this doc. -->
+import IconManufacturer from 'terra-icon/lib/icon/IconManufacturer';
```

Comment:
@eawww is correct. This icon is reserved exclusively for CE Marking, particularly disclosing the Electronic Instructions for Use (commonly abbreviated as eIFU) and Solution Labeling requirements. You are free to use it if you construct the full unit/fragment and use it correctly:  https://wiki.cerner.com/display/UserExperience/About+Solution+Dialog
and https://github.cerner.com/orion/orion-application/tree/master/src/about-box 

Otherwise using this icon should be avoided, as it would be suggesting others can use it for any purpose as well.

Reply:
I will remove all CE icons and use different examples.